Jenkinsfile

```
@@ -3,8 +3,7 @@
  * easy Linux/Windows testing and produces incrementals. The only feature that relates to plugins is
  * allowing one to test against multiple Jenkins versions.
  */
-buildPlugin(useContainerAgent: true, configurations: [
-  [platform: 'linux', jdk: 21],
-  [platform: 'linux', jdk: 17],
+buildPlugin(useContainerAgent: false, configurations: [
+  [platform: 'windows', jdk: 17],
```

Any reason the Java 21 build is being dropped? I think we should keep it if possible.

```suggestion
  [platform: 'windows', jdk: 21],
  [platform: 'windows', jdk: 17],
```